### ISSUE TITLE
Add ingredient analysis for Open Beauty Facts

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#545]
In Open Beauty Facts, we display an ingredient analysis on the web version.
To provide value in the mobile app, we need to display this analysis (once available in the API).

**Ingredients analysis:**
- Without Parabens
- Without Formaldehyde releasers
- With Allergenic fragrances
- Without Silicones
- Without Phenoxyethanol
- Without Triclosan
- Without Cetrimonium bromide
- Without Methylisothiazolinone
- Without Aluminum salts```

**Comment by teolemon** [comment]
API issue for the server: https://github.com/openfoodfacts/openfoodfacts-server/issues/789

**Comment by bamberino** [comment]
Yes, please also list the ingredients what are responsible for the result, e.g.: With Allergenic fragrances: Limonene (incl. link to explaination for the ingredient).

**Comment by teolemon** [comment]
```
en:Parabens
-en:Parabens

en:Formaldehyde releasers
-en:Formaldehyde releasers

en:Allergenic fragrances
-en:Allergenic fragrances

en:Silicones
-en:Silicones

en:phthalates

en:phenoxyethanol
-en:phenoxyethanol

en:triclosan
-en:triclosan

en:cetrimonium bromide
-en:cetrimonium bromide

en:Methylisothiazolinone
-en:Methylisothiazolinone

en:Aluminum salts
-en:Aluminum salts
```

**Comment by teolemon** [comment]
We can do it client side
